shared/test/test_tables.rb

```diff
@@ -0,0 +1,67 @@
+require 'minitest/autorun'
+require 'rack/test'
+require File.expand_path '../../../deployment', __FILE__
+require File.expand_path '../../middleware/channels_api', __FILE__
+require File.expand_path '../../middleware/tables_api', __FILE__
+
+ENV['RACK_ENV'] = 'test'
+
+class TablesTest < Minitest::Unit::TestCase
+
+  def create_channel
```

Comment:
All three test classes in this directory use this same functionality -- may be worth separating out into a shared test lib.